### ISSUE TITLE
Invited Users do not acquire any Role

### ISSUE THREAD

**Issue by alacret** [8base/issues#34]
Upon accepting an invitation the Users sign in with no role

**Comment by andr11111** [comment]
@alacret when the user was invited did you assign roles to it?

**Comment by alacret** [comment]
@andrei-anisimov no, GUEST role is not in the list, I'm assuming that that should be the Role for invited users without role right?

Right now, when I don't assign a Role it gets added without Role, and I need to edit it and assign GUEST or DEVELOPER role

**Comment by andr11111** [comment]
@alacret the Guest role applies to everyone automatically (including unauthenticated use of the API). We don't explicitly display this role for each user, because it applies automatically. Does it make sense?

**Comment by alacret** [comment]
@andrei-anisimov I think that it will be more intuitive to display it. That way you know that that's the default Role.

Does this role does not have access to the API? Because when I add a Developer, he can't do anything until I add the Developer Role

**Comment by andr11111** [comment]
@alacret the Guest role is primarily intended to give access to unauthenticated API calls. By default it doesn't have access to anything. If you give permissions to this role, it means anyone can do it, even without login.
